subprojects/gradle/ci-logger/src/main/kotlin/com/avito/utils/logging/CILoggingFormatter.kt

```diff
@@ -0,0 +1,39 @@
+package com.avito.utils.logging
+
+import java.io.Serializable
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Date
+
+interface CILoggingFormatter : Serializable {
+
+    fun format(message: String): String
```

Comment:
One more important change.
I removed FormattedMessage which hides details and added unneeded abstraction
Now all logging logic is straightforward.
Formatter modifies messages, Destination decides what and where to write